[2024-07-30T15:12:58.306+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-30T15:12:58.312+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kezekte_etl.transform_data manual__2024-07-30T15:12:55.646081+00:00 [queued]>
[2024-07-30T15:12:58.315+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kezekte_etl.transform_data manual__2024-07-30T15:12:55.646081+00:00 [queued]>
[2024-07-30T15:12:58.315+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-30T15:12:58.319+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_data> on 2024-07-30 15:12:55.646081+00:00
[2024-07-30T15:12:58.322+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=427) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-30T15:12:58.323+0000] {standard_task_runner.py:63} INFO - Started process 429 to run task
[2024-07-30T15:12:58.323+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'kezekte_etl', 'transform_data', 'manual__2024-07-30T15:12:55.646081+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/kezekte_etl.py', '--cfg-path', '/tmp/tmp0ju2phag']
[2024-07-30T15:12:58.324+0000] {standard_task_runner.py:91} INFO - Job 42: Subtask transform_data
[2024-07-30T15:12:58.339+0000] {task_command.py:426} INFO - Running <TaskInstance: kezekte_etl.transform_data manual__2024-07-30T15:12:55.646081+00:00 [running]> on host d6d2ff1599d5
[2024-07-30T15:12:58.367+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kezekte_etl' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-07-30T15:12:55.646081+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-30T15:12:55.646081+00:00'
[2024-07-30T15:12:58.368+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-30T15:12:58.377+0000] {python.py:237} INFO - Done. Returned value was: ('398020000166302', 'Кошка', 'Ева', 'Самка', 'Бенгальская кошка (домашняя)', 'KZC300724-0080647', '30.07.2024 20:11:42', '08.03.2022', 'Активный', 'Владелец животного', 'Мечение животного', 'ИП "Рязанова"', 'Ветеринарная клиника', '30.07.2024', 'Атырау г.а.', 'Не вакцинирован', 'Не стерилизован/кастрирован')
[2024-07-30T15:12:58.378+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-30T15:12:58.384+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=kezekte_etl, task_id=transform_data, run_id=manual__2024-07-30T15:12:55.646081+00:00, execution_date=20240730T151255, start_date=20240730T151258, end_date=20240730T151258
[2024-07-30T15:12:58.422+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-30T15:12:58.429+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-30T15:12:58.429+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
